src/solvers/dg_tree/dg_2d.jl

```diff
@@ -984,8 +986,9 @@ end
   return nothing
 end
 
-@inline function mortar_fluxes_to_elements!(surface_flux_values::AbstractArray{<:Any,4}, equations,
-                                            mortar_l2::LobattoLegendreMortarL2, dg::DGSEM, cache,
+@inline function mortar_fluxes_to_elements!(surface_flux_values,
+                                            mortar_l2::LobattoLegendreMortarL2,
+                                            mesh::TreeMesh{2}, equations, dg::DGSEM, cache,
```

Comment:
Why did you change the order of arguments here? Or are there additional changes that I fail to see?

Reply:
I added the argument `mesh` to be able to dispatch properly. I then changed the order to be consistent with `mesh, equations, solver, cache`.

Reply:
Just for reference: We included this in the [style guide](https://trixi-framework.github.io/Trixi.jl/stable/styleguide/)

> If something needs to be specified in more detail for dispatch, put the additional argument before the general one that is specified in more detail. For example, we use `have_nonconservative_terms(equations), equations` and `dg.mortar, dg`.